OsmIntegrator/Extensions/DbConnectionsExtensions.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+using System.Linq;
+using OsmIntegrator.Database.Models;
+using OsmIntegrator.Enums;
+
+public static class DbConnectionsExtensions
+{
+  public static IEnumerable<DbConnection> OnlyActive(this IEnumerable<DbConnection> connections) =>
+    connections
+      .GroupBy(c => new { c.GtfsStopId, c.OsmStopId })
+      .Select(g => g.OrderByDescending(c => c.CreatedAt).FirstOrDefault())
+      .Where(c => c.OperationType == ConnectionOperationType.Added);
```

Comment:
Wydaje mi się, że zapytanie nie zwróci `null` dla poniższego przykładu, a powinno:

Połączenia między przystankiem A (OSM) i B (ZTM) [kolejność chronologiczna]
1. Dodano połączenie ADDED
2. Usunięto połączenie REMOVED
3. Dodano połączenie ADDED
4. Usunięto połączenie REMOVED

Między przystankami nie ma połącznia, bo ostatnie połączenie zostało usunięte. Zapytanie zwróci połączenie nr 3, a powinno zwrócić null.

Reply:
przedyskutowano na SU

Reply:
tak testy przechodzą

`Passed!  - Failed:     0, Passed:    33, Skipped:     0, Total:    33, Duration: 8 s - OsmIntegrator.Tests.dll (net5.0)`